server/setup_flow.go

```diff
@@ -225,7 +225,7 @@ func (p *Plugin) stepServerAddAppLink() flow.Step {
 			"To finish the configuration, we'll need to add and configure an Application Link in your Jira instance.\n" +
 			"Complete the following steps, then come back here to select **Continue**.\n\n" +
 			"1. Navigate to [**Settings > Applications > Application Links**]({{.JiraURL}}/plugins/servlet/applinks/listApplicationLinks) (see _screenshot_).\n" +
-			"2. Keep checked the Atlassian Product Application Type and enter `{{.PluginURL}}` [link]({{.PluginURL}}) as the application link, then select **Create new link**.").
+			"2. Note: For Jira 9.x - Ensure 'Atlassian product' is selected as the Application type and enter `{{.PluginURL}}` [link]({{.PluginURL}}) as the application link, then select **Create new link**.").
```

Comment:
The new screenshot will be invalid for the older Jira versions. Have we considered that? Or is it changed for the cloud part only?